### ISSUE TITLE
Please Read: Thoughts on our Review...

### ISSUE THREAD

**Issue by radeonxray** [edipetres/UFO-blog-post#4]
**Hey Guys!** 

By now our review of your blog should be posted in separate issues, but before you go sharpen your knives and find your recipe for “_Cooking the 2 assholes who didn’t give us a positive review of our blog_”, we wanted to just give you our general thoughts and 2 cents regarding the blog.

We didn’t mean to be tough on your blog because we just wanted to be assholes, but we wanted to give our honest opinion and based on the review criteria provided by Caroline (that to be fair was pretty vague and difficult to interpret).
Because in just about two weeks, you are going to hand-in your blog to be reviewed for your UFO-exam, and so we wanted to provide you with the best feedback as possible, so you get the best grade (: 

The topic that you chose, and the abstract is really good (and the pictures brought a big smile on our faces), but when Zaeem and I got together to compare our notes and review scores, we were pretty much on the same page regarding your blog.

We spent about an hour discussing our individually prepared reviews of your blog and at the end of our review, we may have figured out why we had so many issues with it: 
While you have chosen an exciting and interesting topic, we feel the topic and scope of the blog were simply too wide and big for a 3-4 page blog.
We really missed some more introduction to the various topics, as well as more detailed examples, theory, context and deep dives into each topic. 
Had you tried to focus on one topic of the various subtopics (Like only focusing on AWS and Security, or another combination of AWS + X), you would have had a better opportunity to dive into the topic with better theory, examples, assumptions, designs and alternative solutions.

We had a similar experience with our blog, in which we originally wanted to compare our LSD- experience with TypeScript with JavaScript. That was shot down as being too broad (rightly so), so instead we settled on writing about three subtopics: Static typing, Strict Null Checking and Interfaces. However, as we started writing our blog and finished writing about Static Typing, we had already written one page and then we realized that it was too ambitious to write about all three.
We ended up only writing about Static Typing, in the context of TypeScript and JavaScript.

We would suggest you would refocus your efforts, choose one topic from your wide range of topics and go deep on that. We are confident that you can be successful (: !

**TL:DR**: You chose an interesting topic, but should have selected a single subtopic to discuss, to better include theory, examples, assumptions, designs and alternative solutions, because in your current blog, you’ve spread yourself too thin, thus the low score.

**Comment by edipetres** [comment]
I actually agree that it has become too broad, and we barely touch the surface on many points. We should narrow it more down and dive deeper into that.

Regarding the brutal honesty - I appreciate that you guys have a good intention and that you spent this much time giving a very valuable feedback. We actually agree with most of your criticism and it also gives a fine ground for improvement. Keep up the good work!

**Comment by radeonxray** [comment]
Just had a quick look at your updated/"new" blog and it looks amazing! 
Just a warning though, I did a quick character count (excluding references) and you are atm.  at 11516, close to 2000 over budget. 

It's always tough to cut content, we just spent the past 2 days cutting and slashing over 3000 characters in our own blog, just to get a bit closer to the 9600 limit.

The "big red flag" for me though, is that even by just looking at the blog, it seems obvious that its over budget, so sharpen your knives and start cutting!

**Comment by radeonxray** [comment]
mmm, also great job on the sources/references! Great improvements!
